README.md

```diff
@@ -1,37 +1,37 @@
 <a href="https://lacework.com"><img src="https://techally-content.s3-us-west-1.amazonaws.com/public-content/lacework_logo_full.png" width="600" title="Lacework" alt="Lacework" /></a>
 
-# Lacework Agent Releases
+# Lacework Linux Agent Releases
 
-This Lacework agent GitHub site provides the officially supported Lacework agent releases. From this site, you can download and install Lacework agent releases. 
+This Lacework GitHub site provides the officially supported Lacework Linux agent releases. From this site, you can download and install the Lacework Linux agent. 
 
-# Lacework Agent Documentation
+# Lacework Linux Agent Documentation
 
-For the Lacework Agent Documentation such as information about supported operating systems, how to configure the agent, agent features, Workload dossiers, AWS Fargate, and more, see [Lacework for Workload Security](https://support.lacework.com/hc/en-us/categories/360001044834-Lacework-for-Workload-Security).
+For information about supported Linux operating systems, how to configure the agent, agent features, workload dossiers, AWS Fargate, and more, see the [Lacework Linux Agent Documentation](https://docs.lacework.com/onboarding/supported-operating-systems).
 
 # Releases
 
 In the right frame, click **Releases** to view all the available agent releases.
-Under the **Assets**, all the release download files are listed. These download files contain the agent installers. In addition, each release contains a direct link to the specific agent release notes and the `docker pull` command for pulling down a Lacework docker install image. 
-For all the agent release notes, see [Release Updates](https://support.lacework.com/hc/en-us/categories/360000539793-Release-Updates).
+The **Assets** section for each release lists all the release download files. These files contain the agent installers. In addition, each release contains a link to the specific agent release notes and the `docker pull` command for pulling down a Lacework docker install image. 
+To view the release notes for all Linux agent releases, see [Linux Agent Release Notes](https://docs.lacework.com/releases/category/linux-agent-releases).
 
 # Install Agents
 
-This README.md file provides installation instructions for installing the agent from this Lacework Agent GitHub repository.
-You can also install the agent from the Lacework Console. For a listing of the different agent install methods, see [Agent Install Options](https://support.lacework.com/hc/en-us/sections/360002375733).
+This README.md file provides instructions for installing the agent from this GitHub repository.
+You can also install the agent from the Lacework Console. For information about the different installation methods, see [Linux Agent Installation Methods](https://docs.lacework.com/onboarding/agent-installation-prerequisites).
```

Comment:
do you think we should mention "linux agent" here too? (OR) may be put a note on the top saying agent in this document means "linux agent".

May be it is implicit here, we do not need to mentioned linux agent?